### ISSUE TITLE
Wallet showing negative balance

### ISSUE THREAD

**Issue by goddardnet** [vertcoin-project/one-click-miner-vnext#204]
OCM wallet address showing negative balance in Insight (not cryptoID)

Total received 22
Total sent 350
Final balance -328

cryptoID showing 26 transactions (25 receive, 1 sent)

Insight showing 33 transactions

OCM show spendable as 2.762....

All transactions from last week showing as unconfirmed (15 confirmations at best) (my wallet address  starts with lowercase v)

Insight does not report confirmations to / from addresses starting with lowercase v.....

OCM V2.0 R16

Windows 10 1909
GTX 1650

**Comment by vertiond** [comment]
Is your OCM spendable balance correct?

**Comment by goddardnet** [comment]
I successfully transferred the 2.762... to my wallet address starting VtKNn..... from the OMC client and this shows in CryptoID.

My previous transfers to my address starting vtc1q... were shown as successful in the client (v2.0 R12/R13) with a balance of 18.581 in Insight however the balance does not show in my VTC QT wallet or CryptoID?

My associated wallet address on Insight Vb1T7... shows

Summary confirmed
Total Received     22.458...       - this is what I have mined 
Total sent            353.720....    - not sure where this has come from ????
Final Balance     -331.262...

My OCM Client currently shows a mining balance (spendable) of 0.246... which I've just fired up to check

Is there a way to forward the 18.581... VTC from vtc1q.... to VtKNn.... or to an exchange from Insight?
